dict_crud/check/dict.py

```diff
@@ -0,0 +1,34 @@
+from dict_crud.check import DictStore
+from typing import Dict
+        
+class GetVal(DictStore):
+    _instance = None
+    
+    def __init__(self):
+        if GetVal._instance != None:
+            raise Exception("Singleton class")
+        GetVal._instance = self
+        
+        self._store_data: Dict[str, str] = {}
+        
+    def getInstance():
+        if GetVal._instance == None:
+            GetVal()
+        return GetVal._instance
+        
+    def set(self, key, value):
+        self._store_data.update({key:value})
+    
+    def _get(self, key):
```

Comment:
what is the purpose of this private method here? If it was used other places (other than `get` method) then it would have made sense to have this private `_get` method. But right now it is not making sense.

Reply:
ok i removed